docs/reference/ilm/actions/ilm-searchable-snapshot.asciidoc

```diff
@@ -5,13 +5,14 @@
 Phases allowed: hot, cold, frozen.
 
 Takes a snapshot of the managed index in the configured repository and mounts it
-as a <<searchable-snapshots,{search-snap}>>.
-
-In the frozen phase, the action mounts a <<partially-mounted,partially mounted
-index>>. In other phases, the action mounts a <<fully-mounted,fully mounted
-index>>. If the original index is part of a
+as a <<searchable-snapshots,{search-snap}>>. If the index is part of a
 <<data-streams, data stream>>, the mounted index replaces the original index in
-the data stream.
+the stream.
+
+The `searchable_snapshot` action requires <<data-tiers,data tiers>>. In the
```

Comment:
Could we expand this a bit and say something like:

```
The `searchable_snapshot` action requires a system that uses <<data-tiers,data tiers>> as it will mount the index directly on the corresponding tier using the <<tier-preference-allocation-filter, `index.routing.allocation.include._tier_preference`>> setting.
```
Maybe @jrodewig has a better suggestion here? (I don't particularly like "a system that uses ..." but not sure how else to phrase this).

Reply:
Thanks @andreidan. I pushed some changes to incorporate the `index.routing.allocation.include._tier_preference` setting content.

Reply:
Thanks James